### PR TITLE
Release/release 3.1.0

### PR DESCRIPTION
**Pull request by nvthongswansea**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 3.1.0 (June 30, 2020)
+
+FEATURES:
+* Storage can be increased its speed via UpdateStorag.
```
Comment (itakouna) [inline CHANGELOG.md:6]: Typo

Comment (nvthongswansea) [reply CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+* Support custom HTTP headers.
+* Be able to deal with GS rate limit.
```
Comment (itakouna) [inline CHANGELOG.md:8]: Support gridscale API rate limit or bypass gridscale API rate limit??

Comment (nvthongswansea) [reply CHANGELOG.md:8]: bypass is the one.

Comment (bkircher) [reply CHANGELOG.md:8]: There's a typo `UpdateStorag.` And maybe change wording here `Be able to deal with GS rate limit.` to `Be able to deal with gridscale API rate limiting.`

Comment (nvthongswansea) [reply CHANGELOG.md]: Done.

**Pull request merged**